src/components/suggestions.ts

```diff
@@ -3,17 +3,25 @@ import { Database } from 'sqlite';
 import { openDB } from './db';
 
 // maps from key to readable string
-export const suggestionStates: { [key: string]: string } = {
+export const suggestionStatesReadable: { [key: string]: string } = {
   created: 'Created',
   pending: 'Pending',
   rejected: 'Rejected',
   actionable: 'Actionable',
   accepted: 'Accepted'
 };
 
+export enum suggestionStates {
```

Comment:
nit: we capitalize enum name and remain singular by convention, i.e. `SuggestionState`

Reply:
o